chaosblade-exec-common/src/main/java/com/alibaba/chaosblade/exec/common/model/matcher/BasePredicateMatcherSpec.java

```diff
@@ -23,6 +23,20 @@
  */
 public abstract class BasePredicateMatcherSpec implements MatcherSpec {
 
+    protected boolean noArgs;
+
+    protected boolean required;
+
+    public BasePredicateMatcherSpec() {
+        this.noArgs = false;
+        this.required = false;
+    }
+
+    public BasePredicateMatcherSpec(boolean noArgs, boolean required) {
```

Comment:
I don't think it is necessary to add the constructor. We can only add it in `MethodNameMatcherSpec` class.